transcom/mymove
# [MB-3227] Redux Refactor: PPM (write)

## Description

This PR migrates customer app code that writes PPMs to Redux to the new saga + entities pattern.
- Added API functions for load, create, and patch PPMs
- `loadPPMs` actions are replaced with making the API call `getPPMsForMove` inline, and then storing the response in Redux with `updatePPMs`
- `createPPM` actions are replaced with making the API call `createPPMForMove` inline, and then storing the response in Redux with `updatePPM`
- `updatePPM` actions are replaced with making the API call `patchPPM` inline and then storing the response in Redux with `updatePPM`

This follows the same pattern as the other Redux Refactor (write) changes.

## Reviewer Notes

Pages/flows to test:
- Log in as a service member and create a PPM
- Make changes on the Date and Location, Weight pages and verify they save correctly
- Make changes on the Edit Date and Location page and verify they save correctly
- Verify you can submit the PPM move successfully
- Verify the PPM data loads correctly throughout the customer app
- Verify you can complete the Payment Request flow as a service member

## Setup

To start the app locally:
```
make server_run
make client_run
```

## Code Review Verification Steps


 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3227) for this change
